server.js

```diff
@@ -10,38 +1,47 @@
+const express = require("express");
+var app = express();
+var server = require("http").Server(app);
+var io = require("socket.io")(server);
+const port = 6969;
+
+app.get("/", (req, res) => {
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/CryptoChat-dev/cryptochat/pullRequest?prid=7237627)
```suggestion
  res.sendFile('templates/client_chat.html', { root: __dirname });
```